src/scanner/rule-to-links-mappings.ts

```diff
@@ -29,7 +29,7 @@ export const ruleToLinkConfiguration: DictionaryStringTo<HyperlinkDefinition[]>
     'image-redundant-alt': [BestPractice],
     'input-image-alt': [link.WCAG_1_1_1],
     'object-alt': [link.WCAG_1_1_1],
-    'link-name': [link.WCAG_2_4_4, link.WCAG_4_1_2],
+    'link-name': [/*link.WCAG_2_4_4, link.WCAG_4_1_2*/ BestPractice],
```

Comment:
can we link to issue axe core 2459 in the code here?

Reply:
Added